test/functional/Valhalla/src/org/openj9/test/lworld/StrictFieldGenerator.java

```diff
@@ -62,7 +62,7 @@ private static Class<?> generateTestPutStrictFinalField(String className, boolea
             mv.visitFieldInsn(PUTFIELD, className, fieldName, fieldDesc);
         }
         mv.visitInsn(RETURN);
-        mv.visitMaxs(1, 1);
+        mv.visitMaxs((lateLarval ? 2 : 1), 1);
```

Comment:
This seems unrelated to the subject of this pull request. Please remove or explain.